### ISSUE TITLE
Method Scroll.MakeChildVisible(CanvasObject)

### ISSUE THREAD

**Issue by matwachich** [fyne-io/fyne#3029]
### Is your feature request related to a problem? Please describe:
Example: a Form inside a Scroll ; we Tab-navigate Form Entries, when we arrive at a non visible Entry, the Scroll should follow.

### Is it possible to construct a solution with the existing API?
I don't think so...

### Describe the solution you'd like to see:
Add a method to the Scroll container Scroll.MakeChildVisible(fyne.CanvasObject) that developper can call in Entry.OnFocusGained

**Comment by Jacalz** [comment]
Hmm. Nice idea with focus moving the view. I wonder if that perhaps should be done automatically by the focus manager? cc @toaster

**Comment by toaster** [comment]
@Jacalz, you mean that the focus manager should automatically detect if a (newly) focused element is inside a scroll container and adjust its scrolling accordingly?
Might work.

**Comment by Jacalz** [comment]
> @Jacalz, you mean that the focus manager should automatically detect if a (newly) focused element is inside a scroll container and adjust its scrolling accordingly?

Indeed. That was what I had in mind. Would remove a lot of repetitive code calling the function mentioned above.

**Comment by andydotxyz** [comment]
Sounds cool. Though maybe `ScrollToObject` might fit the existing naming better
